coalib/bearlib/languages/definitions/m4.py

```diff
@@ -0,0 +1,11 @@
+from coalib.bearlib.languages.Language import Language
+
+
+@Language
+class m4:
+    versions = 1.4,
+    extensions = '.m4',
+    comment_delimiter = '#'
+    multiline_comment_delimiters = {'/*': '*/'}
+    string_delimiters = {'`': "'", '"': '"'}
```

Comment:
only "backtick,single-quote" are default start/end, and they are *nested*.

So that cant be stored in this property until we have a solution for https://github.com/coala/coala/issues/5573

https://www.gnu.org/savannah-checkouts/gnu/m4/manual/m4-1.4.18/html_node/Quoted-strings.html

Reply:
Oh I see, seems like `string_delimiters` too needs to be removed atm.